From 63913ffe54b9ee9af2b314b572da46085b34dda8 Mon Sep 17 00:00:00 2001
From: Owen W. Taylor <otaylor@fishsoup.net>
Date: Wed, 10 Nov 2010 22:08:40 +0000
Subject: Pass the correct GDK library when working around Debian libtool

We need to link against gdk/libgdk-x11-3.0.la, not
gdk/x11/libgdk-x11.la, which is the convenience library containing
the backend functions.
---
diff --git a/gtk/Makefile.am b/gtk/Makefile.am
index f8c47fa..36c02ce 100644
--- a/gtk/Makefile.am
+++ b/gtk/Makefile.am
@@ -916,7 +916,7 @@ Gtk_3_0_gir_CFLAGS = \
 		-DGTK_TEXT_USE_INTERNAL_UNSUPPORTED_API
 Gtk_3_0_gir_LIBS = $(gtktargetlib)
 if USE_X11
-Gtk_3_0_gir_LIBS += $(top_builddir)/gdk/x11/libgdk-x11.la
+Gtk_3_0_gir_LIBS += $(top_builddir)/gdk/libgdk-x11-3.0.la
 endif
 Gtk_3_0_gir_FILES = $(introspection_files)
 INTROSPECTION_GIRS += Gtk-3.0.gir
--
cgit v0.8.3.1
